### PR TITLE
Fix/2141  no redict for unknow workspace or unknown file

### PR DESCRIPTION
**Pull request by basilelegal**
#2141 

## Checkpoints

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: frontend/src/container/WorkspaceContent.jsx [hunk 1]
```diff
@@ -100,18 +100,23 @@ class WorkspaceContent extends React.Component {
...
+      if (props.workspaceList.length > 0) wsToLoad = props.workspaceList[0].id
       else return
-    } else wsToLoad = match.params.idws
+    } else wsToLoad = props.match.params.idws
 
-    await this.loadContentList(wsToLoad)
+    try {
+      await this.loadContentList(wsToLoad)
+    } catch (error) {
+      console.log('error while loading content list in workspace content mount')
```
Comment (buxx) [inline frontend/src/container/WorkspaceContent.jsx]: Possibility to have error message in printed log ?

File: frontend/src/container/WorkspaceContent.jsx [hunk 2]
```diff
@@ -134,9 +139,11 @@ class WorkspaceContent extends React.Component {
 
     if (prevState.workspaceIdInUrl !== workspaceId || prevFilter !== currentFilter) {
       this.setState({ workspaceIdInUrl: workspaceId })
-      this.loadContentList(workspaceId)
       this.buildBreadcrumbs()
     }
+    if (!state.contentLoaded) {
+      this.loadContentList(workspaceId).catch(e => console.log('error while loading content list in workspace content update'))
```
Comment (buxx) [inline frontend/src/container/WorkspaceContent.jsx]: Same question about error message here

File: frontend/src/container/WorkspaceContent.jsx (hunk 1 above)
Comment (basilelegal) [reply frontend/src/container/WorkspaceContent.jsx]: You're right, done.

File: frontend/src/container/WorkspaceContent.jsx (hunk 2 above)
Comment (basilelegal) [reply frontend/src/container/WorkspaceContent.jsx]: You're right, done.

**Pull request merged**